arango/formatter.py

```diff
@@ -215,18 +217,24 @@ def format_collection(body: Json) -> Json:
         result["schema"] = body["schema"]
 
     # New in 3.10
-    if body.get("computedValues") is not None:
-        result["computedValues"] = [
-            {
-                "name": cv["name"],
-                "expression": cv["expression"],
-                "overwrite": cv["overwrite"],
-                "computedOn": cv["computedOn"],
-                "keepNull": cv["keepNull"],
-                "failOnWarning": cv["failOnWarning"],
-            }
-            for cv in body["computedValues"]
-        ]
+    if "computedValues" in body:
```

Comment:
This diff changes the function's default behavior. Before, `computedValues` was only present in `results` if `body["computedValues"]`'s value was non-null. A user could assume that `computedValues` is non-null if it's present in `results`, so this change is breaking.

Unless we're planning a major version release next, IMO we should keep the old behavior instead. What do you think?

Reply:
Thanks for pointing that out! This change was made to make the tests pass, which seemed to have been failing before, because we would always expect `computedValues` to appear in the `result`, as long as it appears in the `body`.
For some reason, it was merged with the failing tests. I assume it was due to the fact that the *CI never checked 3.10 before*, it was only validated on 3.7 single-server.
It's possible that the tests were written with the wrong assumptions or the function's behavior isn't well-defined when it comes to null values in `computedValues`.
1. If we're confident that the tests are correct and reflect the intended behavior of our code, we can accept this change. Being new to the project, I used the tests as guidance, so this was more of a bug-fix to me. Note that for all other keys, their value will be reflected in `result`, as long as they are `in` the `body`. This was the only place where I could find this sort of check. It can be `null` in the JSON response, which translates to `None` in `body`, and the tests expect to see the same in `result`.
2. It is also possible that the tests were never adapted to the newly introduced keys. In that case, we should revisit them and align them with the function's current behavior.
What are your thoughts on this?

Reply:
I followed with your suggestion and reverted the way we calculate `computedValues` back to the old behavior.
It is unclear whether or not there should be `None` values in the driver's result. Some attributes are simply copied over to the result, while others, such as `computedValues`, are only processed if they are not `None`. In order to adapt the tests, I had to remove `None` values from both the `result` and the `body` before comparing them. There is a common function used to validate the differences between the request's result body and the expected result after it being processed by the driver: _mock_verify_format_